internal/ws/wsserver_test.go

```diff
@@ -192,16 +192,27 @@ func TestTimeoutProcessing(t *testing.T) {
 	assert.NoError(err)
 
 	c.WriteJSON(&webSocketCommandMessage{
-		Type: "ack",
+		Type:  "ack",
+		Topic: "mytopic",
+	})
+	c.WriteJSON(&webSocketCommandMessage{
+		Type:  "ack",
+		Topic: "mytopic",
 	})
+	c.Close()
 
-	// Confirm we close the connection after the timeout pops
 	for len(w.connections) > 0 {
 		time.Sleep(1 * time.Millisecond)
 	}
 
+	for _, conn := range w.connections {
```

Comment:
I'm also confused about whether `w.connections` is populated at this point. Seems like after the above loop it would be empty, so this would be a no-op? Maybe I'm not following correctly.

Reply:
resolved by above